src/Symfony/Component/Translation/Translatable.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+// Load the global _t() function
+require_once __DIR__.'/Resources/functions/translatable.php';
+
+/**
+ * @author Nate Wiebe <nate@northern.co>
+ */
+final class Translatable
```

Comment:
i tend to believe we should put it in the contracts package.

Reply:
implementing jsonSerializable seems nice (we're translating client-side, so i'd probably expose such objects directly in my API).

Reply:
To confirm, we should add back the interface, and the interface should also extend `\JsonSerializable` or our `final` implementation should?

Reply:
`final class Translatable implements \JsonSerializeable` is fine :)

Reply:
hm on 2nd thought, better keep it a pure object. not sure about the default json format ("messageId", "message_id"?)

Reply:
hm on 2nd thought, better keep it a pure object. not sure about the default json format ("messageId", "message_id"?)

Reply:
Sounds good.